docs/DeveloperGuide.md

```diff
@@ -2,36 +2,524 @@
 
 ## Acknowledgements
 
-{list here sources of all reused/adapted ideas, code, documentation, and third-party libraries -- include links to the original source as well}
+1. Reference to AB-3 Developer Guide
+
```

Comment:
Overall well done! Very organised and neat developer guide. The guide explains the application clearly with sufficient details.